releasenotes/notes/RB-groups-refactor-cb7d54ef96d29324.yaml

```
@@ -1,4 +1,7 @@
 ---
+prelude:
+    Refactor Randomized Benchmarking code to integrate the updated Clifford class
+    :class:`~qiskit.quantum_info.Clifford` and improve the CNOT-Dihedral class.
```

Could we reword this slightly to make it sound more like an introduction, maybe something like:

```suggestion
    The main change made in this release is a refactor of the randomized benchmarking code to
    integrate the updated Clifford class :class:`~qiskit.quantum_info.Clifford` from Terra and to
    improve the CNOT-Dihedral class.
```